taca/analysis/analysis_nanopore.py

```diff
@@ -217,35 +218,66 @@ def process_minion_delivery_run(minion_run):
             
 
 def ont2couch(ont_run):
-    """ Check run vs statusdb. Create or update as needed
+    """ Check run vs statusdb. Create or update run entry as needed.
     """
 
+    run_pattern = re.compile("^(\d{8})_(\d{4})_([0-9a-zA-Z]+)_([0-9a-zA-Z]+)_([0-9a-zA-Z]+)$")
+
     try:
         sesh = NanoporeRunsConnection(CONFIG["statusdb"], dbname="nanopore_runs")        
 
+        if re.match(run_pattern, ont_run.run_id):
+            logger.debug(f"Run {ont_run.run_id} looks like a run directory, continuing.")
```

Comment:
Is there a reason to use `debug` instead of `info`? Just curious 😄

Reply:
Didn't want to clog the log with the default priority :^)